internal/git/branch.go

```diff
@@ -65,9 +82,11 @@ func (r *Repository) LocalBranches(ctx context.Context) ([]LocalBranch, error) {
 		return nil, fmt.Errorf("git branch: %w", err)
 	}
 
-	slices.SortFunc(branches, func(a, b LocalBranch) int {
-		return strings.Compare(a.Name, b.Name)
-	})
+	if sort == "" {
```

Comment:
With the change above, this becomes:
```suggestion
	if opts.Sort == "" {
```